### ISSUE TITLE
cant babel transform requrie ?????

### ISSUE THREAD

**Issue by cleverboy32** [babel/babel#10005]
```
  require([`./docs/${component.name}.md`], res, () => {
                        require([`./docs/components/${component.name}.md`], res);
                    });
```
the error 
![image](https://user-images.githubusercontent.com/17595456/58087764-d148f680-7bf3-11e9-8965-1ee41484ae46.png)

what should i do ? help

**Comment by xtuc** [comment]
Babel will transform your files individually, it's then up to a bundler to concatenate them together (removing the `require`s).

**Comment by nicolo-ribaudo** [comment]
Note that if you are using Babel v7.4.0-7.4.2 with corejs3, it might be caused by a Babel bug which has been fixed in 7.4.3: https://github.com/babel/babel/pull/9754
